code/src/containers/forms/sign-up-form/SignUpForm.test.tsx

```diff
@@ -25,6 +26,16 @@ const mockCloseModal = jest.fn();
   closeModal: mockCloseModal
 });
 
+jest.mock("@/hooks/use-input-visibility/useInputVisibility", () => ({
+  __esModule: true,
+  default: jest
+    .fn()
+    .mockImplementation(
+      jest.requireActual("@/hooks/use-input-visibility/useInputVisibility")
+        .default
+    )
+}));
```

Comment:
Why we need so complex mock here? just `jest.fn()` won't be enough?

Reply:
I needed to make sure that one of the imports was called and at the same time not change its implementation. That is the only way I found that works well. In simpler case `jest.spy` could be used, but it doesn't work in this case, maybe because there are restrictions when using it with imports.

Reply:
Ok, i got it! Unconventional solution! But I recommend adding a comment to it to avoid unpurposly deleting by other developer

Reply:
Added a comment as you suggested.